test/plugins/test_filefilter.py

```diff
@@ -91,31 +94,28 @@ def __create_import_dir(self, count):
         for i in range(count):
             metadata["artist"] = "Artist %d" % (i + 42)
             metadata["track"] = i + 5
-            metadata["title"] = "Tag Title Misc %d" % (i + 1)
+            metadata["title"] = "Tag Track Misc %d" % (i + 1)
             track_file = bytestring_path("track_%d.mp3" % (i + 1))
             dest_path = os.path.join(self.misc_path, track_file)
             self.__copy_file(dest_path, metadata)
             self.misc_paths.append(dest_path)
 
-    def __run(self, expected_lines, singletons=False):
+    def _run(self, expected_lines):
```

Comment:
`__run` has been renamed to `_run`, but then shouldn't `__create_import_dir` also be renamed to `_create_import_dir`?

Reply:
Realised I missed out on removing the import files setup from here. Ended up refactoring this module significantly - the aforementioned method is now gone and the tests should be more clear to follow.